### PR TITLE
Add default styles to FILE template

### LINKED ISSUE #12627: [BUG]  sizing_mode="stretch_both" fails since 3.0.0
### Software versions

Python version      :  3.9.13 (main, Aug 18 2022, 19:15:15) 
IPython version     :  (not installed)
Tornado version     :  6.2
Bokeh version       :  3.0.2
BokehJS static path :  /home/abibok/kisters/repos/service/venv/lib/python3.9/site-packages/bokeh/server/static
node.js version     :  v19.0.1
ERROR: [Errno 2] No such file or directory: 'npm'

### Browser name and version

Google Chrome Version 107.0.5304.110 (Official Build) (64-bit)

### Jupyter notebook / Jupyter Lab version

_No response_

### Expected behavior

Vertical sizing matches the size of the frame, if sizing_mode="stretch_both" is set in a figure

### Observed behavior

Vertical sizing fails, if sizing_mode="stretch_both" is set in a figure. Regardless of the setup, it fails even with the "graph layout example". Just adding that single line, the vertical axis collapses (see the screenshots).

### Example code

```Python
import math


from bokeh.models import Ellipse, GraphRenderer, StaticLayoutProvider
from bokeh.palettes import Spectral8
from bokeh.plotting import figure, show

N = 8
node_indices = list(range(N))
plot = figure(
    title="Graph Layout Demonstration",
    sizing_mode="stretch_both",
    x_range=(-1.1, 1.1),
    y_range=(-1.1, 1.1),
    tools="",
    toolbar_location=None,
)

graph = GraphRenderer()

graph.node_renderer.data_source.add(node_indices, "index")
graph.node_renderer.data_source.add(Spectral8, "color")
graph.node_renderer.glyph = Ellipse(height=0.1, width=0.2, fill_color="color")

graph.edge_renderer.data_source.data = dict(start=[0] * N, end=node_indices)

# create a static layout
circ = [i * 2 * math.pi / 8 for i in node_indices]
x = [math.cos(i) for i in circ]
y = [math.sin(i) for i in circ]
graph_layout = dict(zip(node_indices, zip(x, y)))
graph.layout_provider = StaticLayoutProvider(graph_layout=graph_layout)

# draw quadratic bezier paths
def bezier(start, end, control, steps):
    return [
        (1 - s) ** 2 * start + 2 * (1 - s) * s * control + s**2 * end for s in steps
    ]


xs, ys = [], []
sx, sy = graph_layout[0]
steps = [i / 100.0 for i in range(100)]
for node_index in node_indices:
    ex, ey = graph_layout[node_index]
    xs.append(bezier(sx, ex, 0, steps))
    ys.append(bezier(sy, ey, 0, steps))
graph.edge_renderer.data_source.data["xs"] = xs
graph.edge_renderer.data_source.data["ys"] = ys

plot.renderers.append(graph)

show(plot)
```

### Stack traceback or browser console output

_No response_

### Screenshots

Expected:
image

Observed:

image

### PR DESCRIPTION
**Pull request by mattpap**
fixes #12627

### CODE REVIEW

**Review by bryevdv — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # HTML files
 *.html
+!/src/bokeh/core/_templates/*.html
```
Comment (bryevdv) [inline .gitignore:18]: nice, was not aware of this syntax. Will be useful for thumbnails dir, etc.

**Comment by bryevdv** [comment]
Same two examples failures on a re-run

**Comment by mattpap** [comment]
I confirmed this also fixes dask's issues in jupyter-lab.

**Comment by bryevdv** [comment]
cc @jrbourbeau we should have  a 3.0.3 release next week. There may also be some small usage changes on the dask side as well (e.g. setting a different sizing mode) @mattpap can advise on those specifics.

**Review by jrbourbeau — COMMENTED** [review]
Awesome, thank you both for your work here

**Comment by bryevdv** [comment]
@mattpap do you know if the two test issues are real issues, or a testing artifact of some kind?

**Comment by mattpap** [comment]
Those are real issues and really not that unexpected. I'm investigating what to do about this.

**Comment by mattpap** [comment]
So, this was actually a testing issue. Testing of those examples was broken before, resulting in too small images 8x8 px, due to body margins. In this PR margins were removed and things started fail hard. I don't fully understand why they fail. `Bokeh.index` is missing and thus the test framework can't compute a bbox for image capture. This has something to do with those two examples being animated. For now I skipped them, as we don't do anything with out images anyway.

**Pull request merged**